src/fragments/start/getting-started/flutter/generate-model.mdx

```diff
@@ -8,7 +8,7 @@ With the basic setup complete, you can now model the data your application will
 
 ## Create data model
 
-1. Navigate to the [Data feature in AWS Amplify Studio](https://sandbox.amplifyapp.com/start#datastore) and, with **To-do list schema** selected, click **Create new schema**.
+1. Navigate to the [Data feature in AWS Amplify Studio](https://sandbox.amplifyapp.com/getting-started) and, with **To-do list schema** selected, click **Create new schema**.
 
     ![Create new schema](/images/lib/getting-started/flutter/generate-model-admin-ui-create.png)
 
```

Comment:
Further down on this page, under the **Generate the models locally** section, I think some of this may be outdated as well now that Amplify Studio interface changed:
```
On the Test locally in your app screen, you will be prompted with on-screen instructions
    1. **What type of app are you building?**
        1. Platform: **Cross-platform**
        2. Framework / Language: **Flutter**
        3. Click **Next**
    2. **(Optional) Create a new app**
        1. You should have already created a new Flutter app in a previous step, so go ahead and click **Next**
    3. **Install Amplify CLI to pull the data model**
        1. You should have already installed the Amplify CLI in a previous step
        2. Copy and run the command shown in your project root. This command will initialize your current project with Amplify as well as generate the data models you will be using locally
```
This is no longer accurate. Instead, I think the new steps allow you to essentially skip to the copy command. I suggest we update it to something like:
```
2. On the Local setup instructions screen, you will should see some on-screen instructions
    1. **Local setup instructions for** Flutter (This should already be selected for you)
    2. **Install Amplify CLI to pull the data model**
        1. You should have already installed the Amplify CLI in a previous step
        2. Copy and run the command shown in your project root. This command will initialize your current project with Amplify as well as generate the data models you will be using locally
```

Reply:
This section could definitely be made much simpler.  I have update it to reflect current Amplify Studio Steps.